src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -1625,17 +1618,29 @@
                                Margin="5,5,5,5"
                                RenderTransformOrigin="0.5, 0.5">
                                 <Image.RenderTransform>
-                                    <RotateTransform Angle="90" />
+                                    <RotateTransform Angle="-90" />
```

Comment:
Curious what this is for?

Reply:
That's for rotating a UI element. This is related to the gymnastics I mentioned. Since the rotation of the panel changed directions, this one needs to change as well.